django/db/models/sql/compiler.py

```diff
@@ -40,6 +40,14 @@ def __init__(self, query, connection, using):
         self.klass_info = None
         self._meta_ordering = None
 
+    def __repr__(self):
+        return "<%s query='%s' connection=%s using=%s>" % (
+            self.__class__.__qualname__,
+            self.query,
```

Comment:
I don't think it's worth to include the entire SQL, it should be enough to show a model.